### ISSUE TITLE
[Feature Request] Convert existing subtitles to VTT if doesn't exist

### ISSUE THREAD

**Issue by Nottt** [pannal/Sub-Zero.bundle#376]
That would be a cool feature to have :)

**Comment by pannal** [comment]
Yeah, local subtitle management is a feature planned for one of the next big releases, such as 2.1, 2.2 etc.

It's not super trivial, but I should be able to add existing local subtitles to the item menus, including all of the features.

**Comment by Nottt** [comment]
Thanks :) Don't worry about it :)

At least for UTF-8 encoding and converting existing subs to VTT you might find this useful : 

http://www.nikse.dk/SubtitleEdit/Help#commandline

**Comment by pannal** [comment]
I've incorporated most of the basic stuff SubtitleEdit does, into SZ's modification framework, already, including their OCR fix lists.

**Comment by Smalfinn** [comment]
Is there any update on this feature?

**Comment by pentago** [comment]
+1 for commwnt directly above.
Any updates regarding making dowloaded subtitles UTF-8 encoded?

Thanks!
